build/includes/sdk.mk

```diff
@@ -165,7 +165,8 @@ run-sdk-conformance-test-cpp:
 	$(MAKE) run-sdk-conformance-test SDK_FOLDER=cpp GRPC_PORT=9003 HTTP_PORT=9103
 
 run-sdk-conformance-test-node:
-	$(MAKE) run-sdk-conformance-test SDK_FOLDER=node GRPC_PORT=9002 HTTP_PORT=9102
+	# run with on-by-default (Beta) feature flags enabled
+	$(MAKE) run-sdk-conformance-test SDK_FOLDER=node GRPC_PORT=9002 HTTP_PORT=9102 TESTS=$(DEFAULT_CONFORMANCE_TESTS),$(COUNTS_AND_LISTS_TESTS)
```

Comment:
We would need to execute `npm install` in the sdk folder, e.g. in build-sdk-test.sh the minimum required is
```
cd /go/src/agones.dev/agones/test/sdk/nodejs
npm install --quiet
```
Could we add something like that here? I don't know enough about these makefiles

Reply:
Probably not worth modifying the make command, since the test gets build properly during CI / CD, so this only applies to local testing. I added in a suggested comment change above.